inc/zoo/map/RobinHood_straw.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
```

Comment:
Please let us not use `#pragma once`, the reason is that with proper header guards you can control from test drivers how the headers should look like, I mean overriding the headers actually used from within the test driver